src/browser/modules/Sidebar/ProjectFiles.tsx

```diff
@@ -45,23 +46,12 @@ const ProjectFiles = ({ projectId, scriptDraft, resetDraft }: ProjectFiles) => {
   const [error, setError] = useState('')
 
   function save(inputedFileName: string) {
```

Comment:
I avoided making any more changes to this save func as I think additional work will be occurring here in the next phase potentially, so best to leave it then.